packages/graphql-language-service-interface/src/getAutocompleteSuggestions.ts

```diff
@@ -52,6 +53,34 @@ import {
   objectValues,
 } from './autocompleteUtils';
 
+// TODO@acao,rebornix
```

Comment:
ah i see, yes i dont think we need to take `step` into account, as this context isn't needed in this case